web/satellite/vuetify-poc/src/views/Dashboard.vue

```diff
@@ -160,48 +205,27 @@
             </v-col>
         </v-row>
 
-        <v-row class="d-flex align-center justify-center">
-            <v-col cols="12" md="6">
-                <UsageProgressComponent
-                    icon="globe"
-                    title="Segments"
-                    :progress="segmentUsedPercent"
-                    :used="`${limits.segmentUsed.toLocaleString()} Used`"
-                    :limit="`Limit: ${limits.segmentLimit.toLocaleString()}`"
-                    :available="`${availableSegment.toLocaleString()} Available`"
-                    :cta="!isPaidTier && billingEnabled ? 'Need more?' : 'Learn more'"
-                    @cta-click="onSegmentsCTAClicked"
-                />
+        <v-row align="center" justify="space-between">
+            <v-col cols="12" md="auto">
+                <v-card-title class="font-weight-bold pl-0">Buckets</v-card-title>
+                <v-card-subtitle class="pl-0">Buckets are where you upload and organize your data.</v-card-subtitle>
             </v-col>
-            <v-col cols="12" md="6">
-                <UsageProgressComponent
-                    v-if="isCouponCard"
-                    icon="check"
-                    :title="isFreeTierCoupon ? 'Free Usage' : 'Coupon'"
-                    :progress="couponProgress"
-                    :used="`${couponProgress}% Used`"
-                    :limit="`Included free usage: ${couponValue}`"
-                    :available="`${couponRemainingPercent}% Available`"
-                    :cta="isFreeTierCoupon ? 'Learn more' : 'View Coupons'"
-                    @cta-click="onCouponCTAClicked"
-                />
-                <UsageProgressComponent
-                    v-else
-                    icon="bucket"
-                    title="Buckets"
-                    :progress="bucketsUsedPercent"
-                    :used="`${limits.bucketsUsed.toLocaleString()} Used`"
-                    :limit="`Limit: ${limits.bucketsLimit.toLocaleString()}`"
-                    :available="`${availableBuckets.toLocaleString()} Available`"
-                    cta="Need more?"
-                    @cta-click="onBucketsCTAClicked"
-                />
+            <v-col cols="auto" class="pt-0 mt-0 pt-md-5">
+                <v-btn
+                    variant="outlined"
+                    color="default"
+                    @click="isCreateBucketDialogOpen = true"
+                >
+                    <svg width="16" height="16" class="mr-2" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
is this an svg we will want to use elsewhere? If so, we should store it in vuetify-poc/src/assets with the other svgs

Reply:
yes, thanks for noticing this, we already had this icon as component :)